src/views/login/index.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+
+import GmailLogin from './components/gmail-login'
+import styles from './styles.css'
+
+
+class Login extends Component {
+
+  responseGoogle(googleUser) {
+    const idToken = googleUser.getAuthResponse().id_token
+    console.log({ accessToken: idToken }) // eslint-disable-line no-console
+  }
+
+  render() {
+    return (
+      <main className={styles.main}>
+        <GmailLogin
+          socialId="56778920253-bfeu4u5tub852drphhrbh7iuc7uckojt.apps.googleusercontent.com"
```

Comment:
Could you set this `socialId` value on some config file? It's easier to reutilize/modify later.